supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiInitializationActor.scala

```diff
@@ -131,6 +139,61 @@ class PipelinesApiInitializationActor(pipelinesParams: PipelinesApiInitializatio
     } yield encrypted
   }
 
+  lazy val vpcNetworkAndSubnetworkProjectLabels: VpcAndSubnetworkProjectLabelValues = {
```

Comment:
IMO this should really be a `def vpcNetworkAndSubnetworkProjectLabels(): ... = ???` since this is not a trivial or pure function.